libnavigation-util/src/main/java/com/mapbox/navigation/utils/internal/LoggerProvider.kt

```diff
@@ -9,12 +9,15 @@ import com.mapbox.common.LogWriterBackend
 import com.mapbox.common.LoggingLevel
 import com.mapbox.common.module.provider.MapboxModuleProvider
 
+private const val TAG = "Mapbox"
+private const val CATEGORY = "nav-sdk"
+
 /**
  * Singleton provider of [Logger].
  */
-internal object LoggerProvider {
+object LoggerProvider {
 
-    init {
+    fun initialize() {
```

Comment:
nit: why need separate initialize fun?

Reply:
This is mostly for not running into `ClassNotFoundException`s in tests when accessing to native (so we can mock it).